UnityProject/Assets/Scripts/Systems/Research/LaserPuzzle/ResearchLaserProjector.cs

```diff
@@ -160,6 +160,13 @@ public bool WillInteract(HandApply interaction, NetworkSide side)
 
 	public void ServerPerformInteraction(HandApply interaction)
 	{
+		if (interaction.IsAltClick)
+		{
+			Emitter.TogglePower(!Emitter.IsOn);
+			Chat.AddActionMsgToChat(this.gameObject, $" You toggle The research laser to {Emitter.IsOn} ");
```

Comment:
```suggestion
			Chat.AddActionMsgToChat(this.gameObject, $"You toggle The research laser to {Emitter.IsOn}.");
```

Reply:
Won't this print "true" or "false" to the chat? Doesn't sound like we should print that to the players.